volume/volume_windows.go

```diff
@@ -197,3 +208,9 @@ func ValidMountMode(mode string) bool {
 func ReadWrite(mode string) bool {
 	return rwModes[strings.ToLower(mode)]
 }
+
+// GetCopyMode gets the copy mode from the mode string for mounts. Not supported
+// on Windows - just return false,false
```

Comment:
Will this be supported at some point? (Just curious, as it seems like a big omission if Windows won't have it)